Yoast/javascript
# Add transition words assessment for Arabic

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Implements the transition words assessment for Arabic.
* [Yoast SEO Free] Adds the assessment for transition words in Arabic.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure all tests still pass.
* Add more specs and check that all of them work.

In the plugin:

* Set your environment to Arabic
* Paste in this text containing the transition word `لاسيما`:
`.السفر بالقطار مريح لاسيما على الدرجة الأولى`
* The assessment bullet should be green and read: `Transition words: Well done!`
* The whole sentence should be highlighted.
* Paste in this text containing the transition word `وفي الوقت نفسه`:
` .سأفعل البقالة. وفي الوقت نفسه ، يمكنك الدراسة`
* The assessment bullet should be green and read: `Transition words: Well done!`
* Out of the two sentences, only the second should be highlighted.
* Paste in this text containing the transition word `...كل من ... و`:
`يمكننا الحصول إما على الخبز أو الأرز`
* The assessment bullet should be green and read: `Transition words: Well done!`
* The whole sentence should be highlighted.


## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-518
